Binary files express-pouchdb-orig/.DS_Store and express-pouchdb/.DS_Store differ
Binary files express-pouchdb-orig/lib/.DS_Store and express-pouchdb/lib/.DS_Store differ
diff -uNr express-pouchdb-orig/lib/routes/changes.js express-pouchdb/lib/routes/changes.js
--- express-pouchdb-orig/lib/routes/changes.js	2018-08-09 13:46:54.000000000 -0500
+++ express-pouchdb/lib/routes/changes.js	2018-08-09 13:47:35.000000000 -0500
@@ -34,6 +34,7 @@
       heartbeatInterval = setInterval(function () {
         written = true;
         res.write('\n');
+        if (res.flush) { res.flush(); }   // ADDED NLM TO FIX TIMEOUT
       }, heartbeat);
 
       var cleanup = function () {
